.github/workflows/e2e-connectivity.yaml

```
@@ -0,0 +1,222 @@
+name: NetworkPolicy tests
+
+on:
+  push:
+    branches: [ "main" ]
```

Let's only run this workflow on PRs

```suggestion
  push:
    branches: [ "main" ]
```